### PR TITLE
AAP-31742 - Reconcile chapter 7 of Access management guide

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR reconciles the content for Chapter 7 of the Access management guide with the post code freeze test environment. It also removes the subscription info from the controller docs since it's already covered in the access management docs.

### CODE REVIEW

File: downstream/modules/platform/proc-settings-gw-additional-options.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ From the *Platform gateway settings* page, you can configure additional platform
 . Click btn:[Edit platform gateway settings].
 . You can configure the following *Other settings*:
 +
-* *JWT expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
+* *Jwt expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
```
Comment (sayjadha) [inline downstream/modules/platform/proc-settings-gw-additional-options.adoc:15]: Should this be 'Jwt' just like at the start of the sentence?

File: downstream/modules/platform/proc-settings-gw-security-options.adoc [hunk 2]
```diff
@@ -13,23 +13,23 @@ From the *Platform gateway settings* page, you can configure platform security s
...
 +
 [WARNING]
 ==== 
 If this name is changed, backends must be updated to compensate.
 ====
 +
-* *Platform gateway access token expiration*: How long the access tokens are valid for.
-* *JWT private key*: The private key used to encrypt the JWT tokens sent to back end services. 
+* *Gateway access token expiration*: How long the access tokens are valid for.
+* *Jwt private key*: The private key used to encrypt the JWT tokens sent to back end services.
```
Comment (sayjadha) [inline downstream/modules/platform/proc-settings-gw-security-options.adoc:32]: Same comment, should it be 'Jwt'?

Comment (sayjadha) [inline downstream/modules/platform/proc-settings-gw-security-options.adoc:32]: Suggest 'back end' -> 'backend'

File: downstream/modules/platform/proc-settings-gw-security-options.adoc [hunk 3]
```diff
@@ -38,7 +38,7 @@ This should be a private RSA key and one should be generated automatically on in
 Use caution when rotating the key as it will cause current sessions to fail until their JWT keys are reset.
 ====
 +
-* (Read only) *JWT public key*: The private key used to encrypt the JWT tokens sent to back end services. 
+* (Read only) *Jwt public key*: The private key used to encrypt the JWT tokens sent to back end services. 
 +
```
Comment (sayjadha) [inline downstream/modules/platform/proc-settings-gw-security-options.adoc:42]: Same comment, should it be 'Jwt'?

File: downstream/modules/platform/proc-settings-gw-security-options.adoc [hunk 4]
```diff
@@ -38,7 +38,7 @@ This should be a private RSA key and one should be generated automatically on in
 Use caution when rotating the key as it will cause current sessions to fail until their JWT keys are reset.
 ====
 +
-* (Read only) *JWT public key*: The private key used to encrypt the JWT tokens sent to back end services. 
+* (Read only) *Jwt public key*: The private key used to encrypt the JWT tokens sent to back end services.
```
Comment (sayjadha) [inline downstream/modules/platform/proc-settings-gw-security-options.adoc:41]: Suggest 'back end' -> 'backend'

**Review by sayjadha — COMMENTED** [review]
Few minor edits, rest LGTM.

File: downstream/modules/platform/proc-settings-gw-additional-options.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-settings-gw-additional-options.adoc:15]: No, this is one of those cases where it's wrong in the UI. So I've matched the selections to the UI where I talk about the UI element but left it as correct in the text.

File: downstream/modules/platform/proc-settings-gw-security-options.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-settings-gw-security-options.adoc:42]: Matching the UI here.

**Review by sayjadha — APPROVED** [review]
Changes LGTM.

**Pull request merged**